sjb/generate.py

```diff
@@ -99,11 +99,38 @@ def load_configuration(config_path):
 
     # next, repositories will be synced to the remote VM
     sync_actions = []
+    repo_names=[repository["name"] for repository in job_config.get("sync_repos", [])]
+
+    # determine if test is a pull request
```

Comment:
This does not matter -- if we are running branch builds and build a release branch of repo `x` we still want the correct branch of other repos as well.